src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {

```

Comment:
I think it's to treat empty strings as true? It is confusing. @ZHollingshead If we switch to the more explicit typing above we can probably write some clearer logic here as well.